### DISCUSSION TITLE
Portal Invite Only

### DISCUSSION THREAD

**Discussion by devilmind81** [mediacms-io/mediacms#629] [Q&A]
Hi,
I would like to configure my MediaCMS instance:
1. to be a private portal, so only the registered user can view my videos. 
2. to allow an users only to register itself if receive an invitation from another user via email and click on the invitation link.

For the 1 point I understood from documentation that  that I could make the portal workflow public, but at the same time set GLOBAL_LOGIN_REQUIRED = True so that only logged in users can see content.

But how can I achieve the point 2? Is it possible?

Thanks a lot in advance!

**Comment by grubinstein** [comment]
This is explicitly stated as a capability but I can't seem to find a way to do it. Very interested in an answer to this!
